src/color.js

```diff
@@ -102,10 +102,10 @@ function seqColorScale(colors) {
   };
 }
 
-export var scaleSeqBlu = seqColorScale(["#DDE9FE", "#3B76B3", "#343F4D"]);
-export var scaleSeqRed = seqColorScale(["#FEECEC", "#CC6171", "#4D353A"]);
-export var scaleSeqGrn = seqColorScale(["#D2DFDE", "#4A807C", "#2C3C3F"]);
-export var scaleSeqBrn = seqColorScale(["#E9DFD6", "#A67D5A", "#4C3735"]);
+export var scaleSeqBlu = seqColorScale(["#3431DE", "#0A8DF6", "#23C3F1"]);
```

Comment:
@squiles Do you think we should further isolate the colors into variables to make repetition easier to follow? Would be nice in this config, but on the other hand they do not change often and the information is lost for debugging anyway.

Reply:
I guess for us it would be a nice thing to have in this file as it improves readability. Will include it